rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -349,14 +348,26 @@ public TaskResult getTaskResult(String jobId, String taskName) throws NotConnect
     }
 
     @Override
-    public TaskResult getTaskResult(JobId jobId, String taskName) throws NotConnectedException,
-            UnknownJobException, UnknownTaskException, PermissionException {
+    public TaskResult getTaskResult(JobId jobId, String taskName)
+            throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
         return getTaskResult(jobId.value(), taskName);
     }
 
     @Override
-    public String getTaskServerLogs(String arg0, String arg1) throws UnknownJobException,
-            UnknownTaskException, NotConnectedException, PermissionException {
+    public List<TaskResult> getTaskResultByTag(JobId jobId, String taskTag)
+            throws NotConnectedException, UnknownJobException, PermissionException {
+        throw new UnsupportedOperationException();
```

Comment:
I think because this method should not be exposed to the CLI ? @sandrineBeauche ?